src/Promitor.Tests.Unit/Configuration/RuntimeConfigurationUnitTest.cs

```diff
@@ -391,7 +392,7 @@ public async Task RuntimeConfiguration_HasConfiguredCollectorUriInOpenTelemetryC
             // Arrange
```

Comment:
Let's use InlineData to inject 2 protocol types to make sure it works with different versions and not grpc which can be the default